README.md

```diff
@@ -79,11 +75,16 @@ To start a stream (a thread that reads messages from Kafka), add this to your co
     [message]
     (println message)
     :success)
+    
+(def handler-fn
+    (-> main-fn
+      (middleware/protobuf->hash ProtoClass :stream-id)))
```

Comment:
`ProtoClass` is well-understood, but we can use code comments in this sample to explain it with an example. Something like this:

`;; Here ProtoClass refers to the fully qualified name of the Java class which the code is using to de-serialize the message.`

Reply:
done